netbsd/NetBSDProcess.h

```diff
@@ -0,0 +1,36 @@
+#ifndef HEADER_NetBSDProcess
+#define HEADER_NetBSDProcess
+/*
+htop - NetBSDProcess.h
+(C) 2015 Hisham H. Muhammad
+(C) 2021 Santhosh Raju
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include <stdbool.h>
+
+#include "Object.h"
+#include "Process.h"
+#include "Settings.h"
+
+
+typedef struct NetBSDProcess_ {
+   Process super;
+} NetBSDProcess;
+
+#define Process_isKernelThread(_process) (_process->pgrp == 0)
+
+#define Process_isUserlandThread(_process) (_process->pid != _process->tgid)
```

Comment:
Cf. note on thread handling. These are also obsolete. Implementations should use the `isUserThread` and `isKernelThread` fields from the global `Process` structure instead. Cf. #388 for details.

Reply:
Should be updated in 237e66c